packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1351,7 +1351,7 @@ export const getAccessibleDataObject = (
 
 type LocaleStringDataProps = number | string | Date | undefined;
 export const convertToLocaleString = (data: LocaleStringDataProps, culture?: string): LocaleStringDataProps => {
-  if (!data) {
+  if (data === undefined || data === null || Number.isNaN(data)) {
```

Comment:
Can you add a unit test for these cases - data = false, data = empty string and data = 0

Reply:
Ok

Reply:
Couldn't add test for null as the function doesn't accept null as an input

Reply:
<img width="349" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/ce25b80a-0ed9-42e5-b56e-603a0f545bef">
@AtishayMsft

Reply:
can you add UTss for remaining cases. 
data = false/False
data = "Hello world"
data = "123"
data = ""
data = " "
data = null. You can set the variable being passed as null.

Reply:
Ok

Reply:
<img width="553" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/88e59bd1-1165-4b3b-b6b4-b74c7e1b3ed2">

Setting the variable null isn't allowed by type LocaleStringDataProps 

<img width="382" alt="image" src="https://github.com/microsoft/fluentui/assets/35366351/274aa6da-751c-4f51-80f1-94d5da83b907">